_docs/concepts/what-is-istio/overview.md

```diff
@@ -1,29 +1,34 @@
 ---
 title: Overview
-overview: Provides a broad overview of what problems Istio is designed to solve.
+overview: Provides a conceptual introduction to Istio, including the problems it can solve and its high-level architecture.
           
 order: 15
 
 layout: docs
 type: markdown
 ---
 
-This page introduces Istio: an open platform to connect, manage, and secure microservices.
+This document introduces Istio: an open platform to connect, manage, and secure microservices. Istio provides an easy way to create a network of deployed services with load balancing, service-to-service authentication, monitoring, and more, without requiring any changes in service code. You add Istio support to services by deploying a special sidecar proxy with your service that proxies all network communication between microservices, configured and managed using Istio's control plane functionality.
```

Comment:
'proxy that proxies' doesn't sound too good. Replace 'proxies' with 'intercepts'